notebook/tests/selenium/test_display_isolation.py

```diff
@@ -73,16 +79,20 @@ def isolated_svg(notebook):
         content="display_svg(SVG(s1), metadata=dict(isolated=True))")
     notebook.add_and_execute_cell(
         content="display_svg(SVG(s2), metadata=dict(isolated=True))")
+    wait_for_tag(notebook.browser, "iframe")
     iframes = notebook.body.find_elements_by_tag_name("iframe")
```

Comment:
Here we're waiting for an `iframe` tag to exist, and then getting a list of iframes, which must contain 2 iframes. I wonder if the list is sometimes captured when only one is on the page. Can we figure out a way to wait for two, or to wait for the one we know will appear second?

Reply:
hmm true. I can probably give the iframes ids and wait for both of the ids to be safe.

Reply:
setting ids turned out to be non-trivial but I think I can probably write a custom `wait_for_n` selenium expectation.